app/src/components/learningJourney/CourseContainer.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import AddCourseButton from "./AddCourseButton";
+import CourseCard from "./CourseCard";
+
+export default function CourseContainer() {
+
+    // Call Get all Courses for LJ API here
+    const courseData = [
```

Comment:
Define mock data outside of component.

Also:
1. Can create the FE function to call the BE API if not yet create -> Then link it into this function
2. Ensure that the data mapping of BE API to FE is the same -> Aka BE gives snakecase, FE uses camel. Lets define some common data notation and settle on a finalised schema.

Reply:
removed the mock courseData here and used the mock data in utils/mocks.js. As for the FE functions to call the BE API, i will do in a separate branch after this branch and Ailys's LJ BE branch is merged